src/sfr.hpp

```diff
@@ -148,9 +148,13 @@ namespace sfr {
         extern std::deque<int> mode_history;
     } // namespace mission
     namespace burnwire {
+        // OP Code 4444
+        extern bool fire;
```

Comment:
I'm assuming the movement of this out of an SFRField is to prevent someone using an SFR Override here and then henceforth bypassing the is_valid() constraints on arm and fire?

Reply:
Yes!